projects/batfish/src/org/batfish/grammar/cisco/Cisco_bgp.g4

```diff
@@ -248,11 +261,21 @@ local_as_bgp_tail
    )* NEWLINE
 ;
 
+maximum_ecmp_paths
```

Comment:
What is this for? I only see you using it in null_bgp_tail, where I don't want to see further references to parser rules. It seems you could just add '(ECMP DEC)?' to the maximum_paths_bgp_tail rule.